packages/riipen-ui/src/components/FormGroup.test.jsx

```diff
@@ -0,0 +1,81 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import FormGroup from "./FormGroup";
+
+describe("<FormGroup>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<FormGroup />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  describe("snapshot test", () => {
+    it("renders correctly", () => {
+      const wrapper = mount(<FormGroup />);
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+  });
```

Comment:
We should probably make this consistent with however we choose to do these. See this PR https://github.com/riipen/ui/pull/124#pullrequestreview-580973586.